### PR TITLE
Sort headers, fix overflowing, copy curl

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Comment by xepozz** [comment]
image

- Copy button in the curl section
- Sort headers

image

- Fixed long size of the paths / classes

File: src/Renderer/HtmlRenderer.php [hunk 1]
```diff
@@ -347,17 +347,19 @@ public function renderRequest(ServerRequestInterface $request): string
...
+        ksort($headers);
 
+        foreach ($headers as $name => $values) {
             foreach ($values as $value) {
                 $output .= "$name: $value\n";
             }
         }
 
-        $output .= "\n" . $request->getBody() . "\n\n";
+        if (!empty($request->getBody())) {
```
Comment (vjik) [inline src/Renderer/HtmlRenderer.php:360]: Before empty check need to get body as string.

File: src/Renderer/HtmlRenderer.php [hunk 2]
```diff
@@ -377,7 +379,7 @@ public function renderCurl(ServerRequestInterface $request): string
             return $this->htmlEncode('Error generating curl command: ' . $e->getMessage());
```
Comment (vjik) [inline src/Renderer/HtmlRenderer.php:379]: Not need html encode also

File: templates/development.php [hunk 3]
```diff
@@ -107,12 +107,25 @@
...
+            <h2>Request info</h2>
+            <div class="body">
+                <?= $requestInfo ?>
+            </div>
         </div>
     <?php endif ?>
     <?php if ($request && ($curlInfo = $this->renderCurl($request)) !== 'curl'): ?>
         <div class="request">
-            <?= $curlInfo ?>
+            <textarea id="clipboard"><?= $curlInfo ?></textarea>
```
Comment (vjik) [inline templates/development.php:118]:
```suggestion
            <textarea id="clipboard"><?= $this->htmlEncode($curlInfo) ?></textarea>
```

**Review by vjik — CHANGES_REQUESTED** [review]
Add info to changelog.

**Review by vjik — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 ## 3.2.2 under development
 
-- no changes in this release.
+- Enh #112: Add copy cURL button, fix UI (@xepozz)
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- Enh #112: Add copy cURL button, sort request headers, fix UI (@xepozz)
```

**Pull request merged**